eng/WpfArcadeSdk/tools/CodeGen/DesignTimeTextTemplating.targets

```diff
@@ -0,0 +1,98 @@
+<Project>
+  <ItemGroup>
+    <Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />
+  </ItemGroup>
+  <!-- Including the Microsoft.TextTemplating.targets requires us to manually import Sdk.Targets because
+       it needs to be imported after Sdk.targets. This causes the BuildDependsOn variable,
+       which is modified by the T4 targets, to be overwritten, so the TransformAll target 
+       doesn’t run before the Build target. -->
+  <!-- <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" /> -->
+  <PropertyGroup>
+    <!-- Get the Visual Studio version: -->
+    <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">16.0</VisualStudioVersion>
+    <!-- Keep the next element all on one line: -->
+    <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
+    <!-- This is what will cause the templates to be transformed when the project is built (default is false) -->
+    <TransformOnBuild>true</TransformOnBuild>
+    <!-- Set to true to force overwriting of read-only output files, e.g. if they're not checked out (default is false) -->
```

Comment:
re "_they're not checked out_"

Does this matter in Git repos?

Reply:
I'll plead guilty that i had gotten this bit from a blog post about integrating design time tt in an sdk style project :)

I think you are right that this is probably not necessary, or at least the comment is definitely not relevant for us.